### ISSUE TITLE
Shouldn't be able to start Openboard twice?

### ISSUE THREAD

**Issue by MarkWllms** [OpenBoard-org/OpenBoard#74]
If you launch Openboard on Windows 7 when the software is already open, a second copy starts. This may be undesirable behaviour for a full screen app.

**Comment by freeheel3r** [comment]
You're right, this shouldn't happen (there is code that's supposed to prevent it actually).
I also observe this on Windows 10.
